docker/runtime/backup-init.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+set -e
+
+BACKUP_PATH="/backup"
+
+# Make sure that required directories exist
+mkdir -p "${BACKUP_PATH}"
+mkdir -p "/etc/crontabs"
+chown git:git /backup
+chmod 2770 /backup
+
+# [string] BACKUP_INTERVAL   Period expression
+# [string] BACKUP_RETENTION  Period expression
+if [ -z "${BACKUP_INTERVAL}" ]; then
+	echo "Backup disabled: BACKUP_INTERVAL has not been found" 1>&2
+	exit 1
+fi
+
+if [ -z "${BACKUP_RETENTION}" ]; then
+	echo "Backup retention period not defined - default value used: 7 days" 1>&2
```

Comment:
```suggestion
	echo "Backup retention period is not defined, default to 7 days" 1>&2
```

Reply:
Sorry about that. I always tend to write very generic, robot-alike messages.